### ISSUE TITLE
Relax rule: Do not enforce camelcase style for global variables (camelcase)

### ISSUE THREAD

**Issue by mcdado** [standard/standard#1474]
**What version of this package are you using?**
14.3.1

**What operating system, Node.js, and npm version?**
macOS 10.15.2, Node.js v10.18.1, npm 6.13.4

**What happened?**
In a JS file for the web, that interacts with a page where there's a variable defined in a template (which comes from an open source e-commerce CMS — changing it would break core functionality because other code expects it) I declare a "global" variable `id_product`. Using in the code triggers the `camelcase` rule, but I cannot do anything about it.

```js
/* global $, id_product [...] */

var productId = id_product // Identifier 'id_product' is not in camel case. (camelcase)
```

**What did you expect to happen?**
Since it's a global, stylistic rules shouldn't apply. Is it possible to do it with eslint? I just checked my previous configuration and I had camelcase disabled, probably for this reason.

**Are you willing to submit a pull request to fix this bug?**
Yes.

**Comment by mightyiam** [comment]
https://eslint.org/docs/rules/camelcase

I don't see anything regarding globals. Perhaps take a look at ESLint source code; the tests for the camelcase rule; whether there's a test for ignoring globals.

**Comment by mcdado** [comment]
Thank you, sorry I didn't think hard enough about it that it should be an upstream issue... and it seems that there's not much interest there: https://github.com/eslint/eslint/issues/11716

**Comment by mcdado** [comment]
https://github.com/eslint/eslint/pull/12782 🤞

**Comment by mcdado** [comment]
> https://eslint.org/docs/rules/camelcase
> 
> I don't see anything regarding globals. Perhaps take a look at ESLint source code; the tests for the camelcase rule; whether there's a test for ignoring globals.

Well, I landed the feature in eslint, and it got released in [7.4](https://github.com/eslint/eslint/releases/tag/v7.4.0).

Whenever the upgrade will come it would be nice for this option to be included 😊

**Comment by feross** [comment]
This will ship in `standard` 16
